### PR TITLE
Use NPM Scripts to Run Build, Dev, and Deploy Workflows

### PR DESCRIPTION
**Pull request by luzmcosta**
Adds and documents new npm scripts, specifically targeting the build, dev, and deploy workflows.
- Run cannery-result.js via the new `npm run prebuild` command
- Deploy via the new `npm run deploy` command
- Quickly set up a development environment using the new `npm run dev` command

Note, any command prefixed with "pre" will run as part of the command it prefixes, so running `npm run build` runs `npm run prebuild` first.

### CODE REVIEW

**Comment by henryrizzi** [comment]
Noyce!

File: README.md [hunk 1]
```diff
@@ -15,20 +15,28 @@ git clone git@github.com:tym-oao/nerium-static.git
...
-## Deployment to Netlify
+## Deploy
 
-Assumes `netlifyctl` is installed locally
+Scripts currently assume `netlifyctl` is installed locally:
 
 ``` sh
 npm run build
-netlifyctl deploy
+npm run deploy
```
Comment (tym-xqo) [inline README.md:41]: ❤️❤️❤️

File: src/scripts/cannery-result.js [hunk 2]
```diff
@@ -2,38 +2,66 @@
...
-          JSON.stringify(rows),
-          () => {
-            console.log('done!')
-      })
+        if (err) {
+          // fallback to sqlite for build/test without Postgres connection
+          console.log('Warning: no Postgres connection, trying sqlite...')
+          const sqlite3 = require('sqlite3')
+          lite = new sqlite3.Database(':memory:')
+          // Prepend `:` to parameter keys for sqlite 😠
```
Comment (luzmcosta) [inline src/scripts/cannery-result.js]: haha. On the upside, emojis render really nicely in comments. 😁

File: src/scripts/cannery-config.js [hunk 3]
```diff
@@ -1,7 +1,8 @@
 const CANNERY_CONFIG = {
-    queryName: 'alpha-eu-revenue-monthly-summary',
-    queryParams: {},
-    outputName: 'alpha-eu'
+    queryName: 'test',  // required
+    queryParams: {greeting: 'yo'},  // optional (but must match query to run)
+    // Uncomment below if you want output file name different than queryName
```
Comment (luzmcosta) [inline src/scripts/cannery-config.js:4]: 👍

**Review by luzmcosta — COMMENTED** [review]
Scripts pass checks. 💯 The only thing left to prove is the alpha-eu.json is updated based on configs pointing to the warehouse.

**Comment by henryrizzi** [comment]
Still not able to hit the table necessary for checking the data, but at least I know the error that I'm getting is not connection related, just perm related. so I feel comfortable 👍 this PR.

**Comment by henryrizzi** [comment]
perm issues resolved, this is looking 🔥

**Pull request merged**